tests/feature/test-core-filters.php

```diff
@@ -258,4 +258,33 @@ public function test_get_author_display_name_from_user(): void {
 		$this->assertNotSame( '', $display_name );
 		$this->assertSame( $author_name, $display_name );
 	}
+
+	/**
+	 * Test that the core author block's author name can be filtered using a byline author.
+	 */
+	public function test_replace_author_block_author(): void {
+		global $post;
+
+		// Create a WP user to be the post author.
+		$user = static::factory()->user->create_and_get([
+			'display_name' => 'Author Name',
+			'description'  => 'This is a test author bio.',
+		]);
+
+		// Create a post with the author block.
+		$post = static::factory()->post->create_and_get([
+			'post_author'  => $user->ID,
+			'post_content' => '<!-- wp:post-author {"showBio":true,"byline":"Here is my byline","isLink":true,"linkTarget":"_blank"} /-->',
```

Comment:
is `byline` a wp core attribute?

Reply:
Yep! I double checked by deactivating the Byline Manager plugin.